### PR TITLE
add: service docs readme

### LINKED ISSUE #1395: [Improvement]: Unify service doc comments and crate docs with README files
### Describe the improvement

Our service crates' documentations were recently updated so that the crate documentation
```rs
//! Shuttle service integration for the Axum web framework.
//!
//! ## Example
//!
//! ```rust,no_run
//! use axum::{routing::get, Router};
//!
//! async fn hello_world() -> &'static str {
//!     "Hello, world!"
//! }
//!
//! #[shuttle_runtime::main]
//! async fn axum() -> shuttle_axum::ShuttleAxum {
//!     let router = Router::new().route("/", get(hello_world));
//!
//!     Ok(router.into())
//! }
//! ```
```
is very similar to the return type's documentation
```rs
/// Return type from the `[shuttle_runtime::main]` macro for a Axum-based service.
///
/// ## Example
///
/// ```rust,no_run
/// use axum::{routing::get, Router};
///
/// async fn hello_world() -> &'static str {
///     "Hello, world!"
/// }
///
/// #[shuttle_runtime::main]
/// async fn axum() -> shuttle_axum::ShuttleAxum {
///     let router = Router::new().route("/", get(hello_world));
///
///     Ok(router.into())
/// }
/// ```
```

It would be better to only have one unified version of this doc in the crate's README.md file, and use `#![doc = include_str!("../README.md")]` and `#[doc = include_str!("../README.md")]` on the crate and the return type respectively.

This can be done for every service in the `services/` dir.

---

Bonus task: Update the Cargo package descriptions if needed

--- 

Feel free to comment on this issue if you want to work on it!

### PR DESCRIPTION
**Pull request by AvaterClasher**
## Description of change

Added a README.md for the doc comments in the shuttle-actix-web crate.
Modified the crate to include the README.md during doc compilation.

Closes #1395

### CODE REVIEW

**Comment by AvaterClasher** [comment]
@jonaro00 Can you please take a look if this is how you intended it to be done ?

File: services/shuttle-actix-web/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ #[shuttle_runtime::main]
+ async fn actix_web() -> ShuttleActixWeb<impl FnOnce(&mut ServiceConfig) + Send + Clone + 'static> {
+     let config = move |cfg: &mut ServiceConfig| {
+         cfg.service(hello_world);
+     };
+
+     Ok(config.into())
+ }
+ 
+````
```
Comment (jonaro00) [inline services/shuttle-actix-web/README.md:23]: The codeblock should have 3 backticks, and not be indented with 1 space, and I think we still need `,no_run` to not trigger them in doc tests.

**Review by jonaro00 — COMMENTED** [review]
Yes, looks great apart from this detail:

**Review by jonaro00 — APPROVED** [review]
LGTM! Don't worry about the serenity CI fail.

**Comment by AvaterClasher** [comment]
@jonaro00 How does one get the contributer role in discord ?

**Comment by jonaro00** [comment]
You can ask in #contributors and I'll tell admins to add you and other new contributors.

**Pull request merged**